### PR TITLE
Degree profile updates

### PR DESCRIPTION
**Pull request by jmbarne3**
Enhancements:
* Added plancode and subplan code.
* Added GF filter to make plancode/subplancode accessible for export.
* Updated Degree Single template
* Added Tuition fields.

### CODE REVIEW

File: single-degree.php [hunk 1]
```diff
@@ -63,6 +86,38 @@
 					</ul>
 					<?php endif; ?>
 
+					<?php if ( $has_tuition ) : ?>
+					<h3 class="highlighted"><span>Tuition and Fees</span></h3>
```
Comment (rjucf) [inline single-degree.php]: Is the span tag here necessary?

Comment (jmbarne3) [inline single-degree.php]: Yes. That's how the highlighted headers work in this theme.

File: single-degree.php [hunk 2]
```diff
@@ -43,8 +45,29 @@
 		<?php if ( $sidebar_enabled && $has_sidebar_content ): ?>
 			<div class="row">
 				<div class="col-md-8">
+					<!-- Degree details + credit hours -->
+					<div class="row">
+						<div class="col-md-12 col-sm-12">
+							<dl class="degree-details row">
+								<?php if ( count( $post->taxonomies['program_types'] ) > 0 ) : ?>
+									<dt class="col-md-3 col-sm-3">Program Type:</dt>
+									<dd class="col-md-9 col-sm-9"><?php echo $post->taxonomies['program_types'][0]->name; ?></dd>
```
Comment (cjg89) [inline single-degree.php]: Does accessing `$post->taxonomies['program_types'][0]->name;` directly not throw a notice or warning in dev?  I feel like we've had issues with accessing object properties within an array directly like this before depending on the PHP version running (correct me if I'm thinking of something else)

Comment (jmbarne3) [inline single-degree.php]: I probably should update it... It's not a problem in PHP 5.4, which is what I think we're running in dev now.

**Pull request merged**